pkgs/applications/version-management/gitlab-shell/remove-hardcoded-locations.patch

```diff
@@ -1,23 +1,33 @@
-diff --git a/go/internal/config/config.go b/go/internal/config/config.go
-index c57b4de..88cfc95 100644
---- a/go/internal/config/config.go
-+++ b/go/internal/config/config.go
-@@ -27,7 +27,7 @@ func New() (*Config, error) {
- 	}
- 	cfg.RootDir = dir
+diff -Nuar gitlab-shell.orig/go/internal/config/config.go gitlab-shell/go/internal/config/config.go
+--- gitlab-shell.orig/go/internal/config/config.go	2018-10-29 11:18:37.454033228 +0100
++++ gitlab-shell/go/internal/config/config.go	2018-10-29 11:53:35.352777503 +0100
+@@ -35,7 +35,7 @@
+ }
  
--	configBytes, err := ioutil.ReadFile(path.Join(cfg.RootDir, configFile))
-+	configBytes, err := ioutil.ReadFile("/run/gitlab/shell-config.yml")
- 	if err != nil {
- 		return nil, err
- 	}
-diff --git a/lib/gitlab_shell.rb b/lib/gitlab_shell.rb
-index 1452f95..2b40327 100644
---- a/lib/gitlab_shell.rb
-+++ b/lib/gitlab_shell.rb
-@@ -180,7 +180,8 @@ class GitlabShell
-     end
+ func NewFromDir(dir string) (*Config, error) {
+-	return newFromFile(path.Join(dir, configFile))
++	return newFromFile("/var/run/gitlab/shell-config.yml")
```

Comment:
Should this be `/run` without `/var`?

Reply:
Probably. /var/run is a symlink to /run on my system, didn't notice that before.